src/cli.js

```diff
@@ -224,17 +260,22 @@ async function signInToHeroku() {
  */
 async function signInToThirdPartyTools(configs) {
   const credentials = {};
+  let herokuConfigs = {};
+  const toolOptions = {};
   if (configs.tooling.sourceControl === constants.github.name) {
     const githubCredentials = await signInToGithub();
     credentials.github = githubCredentials;
   }
 
   if (configs.tooling.deployment === constants.heroku.name) {
-    const herokuCredentials = await signInToHeroku();
-    credentials.heroku = herokuCredentials;
+    herokuConfigs = await setupHeroku(configs);
+    credentials.heroku = herokuConfigs.herokuCredentials;
+    toolOptions.appName = herokuConfigs.appName;
```

Comment:
Should this be `herokuAppName` to disambiguate?

Reply:
Comment below